### PR TITLE
raw node inventory history now generated from Kafka data; first round…

### PR DESCRIPTION
**Pull request by cmcheung-UCS**
… of clean up.

Signed-off-by: Ming Cheung <chi.m.cheung@intel.com>

### CODE REVIEW

File: inventory/src/main/java/com/intel/dai/inventory/api/es/RawNodeInventoryIngester.java [hunk 1]
```diff
@@ -48,29 +58,33 @@ String constructAndIngestNodeInventoryJson(FruHost fruHost) throws DataStoreExce
...
+    }
 
-            try {
-                inventoryApi_.addDimm(hostname, hostname + "_" + locator,
-                        "A", sizeMB, locator, null, serial,
-                        doc_timestamp * 1000000L, "INVENTORY", -1);
-            } catch (DataStoreException e) {
-                log_.error(e.getMessage());
-            }
+    private void migrateInventoryDataToPOTable(String locator, String dimmJson, String hostname, long doc_timestamp) {
```
Comment (avincigu) [inline inventory/src/main/java/com/intel/dai/inventory/api/es/RawNodeInventoryIngester.java:69]: I don't think we want to use the PO on the name of these functions, may I suggest maybe just migrateInventoryData or processRawInventoryData?

File: inventory/src/main/java/com/intel/dai/inventory/api/es/RawNodeInventoryIngester.java [hunk 2]
```diff
@@ -140,14 +154,18 @@ public void ingestInitialNodeInventoryHistory() {
...
-                String nodeInventoryJson = constructAndIngestNodeInventoryJson(fruHost);
-                inventoryApi_.addFru(fruHost.hostname, fruHost.timestamp * 1000000L, nodeInventoryJson, fruHost.boardSerial, fruHost.rawIbBios);
+                constructAndIngestNodeInventoryJson(fruHost);
             } catch (DataStoreException e) {
                 log_.error("DataStoreException: %s", e.getMessage());
             }
         }
     }
 
+    private void migrateInventoryDataToPOTable(FruHost fruHost, String nodeInventoryJson) throws DataStoreException {
```
Comment (avincigu) [inline inventory/src/main/java/com/intel/dai/inventory/api/es/RawNodeInventoryIngester.java:164]: Same as above

**Review by avincigu — CHANGES_REQUESTED** [review]

File: inventory/src/main/java/com/intel/dai/inventory/api/es/RawNodeInventoryIngester.java (hunk 1 above)
Comment (cmcheung-UCS) [reply inventory/src/main/java/com/intel/dai/inventory/api/es/RawNodeInventoryIngester.java:69]: I will call it standard instead.

**Review by avincigu — APPROVED** [review]

File: inventory/src/main/java/com/intel/dai/inventory/api/es/RawNodeInventoryIngester.java (hunk 1 above)
Comment (paul-amonson) [reply inventory/src/main/java/com/intel/dai/inventory/api/es/RawNodeInventoryIngester.java]: This inventory API is not going to be in PO at all.

**Review by paul-amonson — APPROVED** [review]

**Pull request merged**